src/Nethermind/Nethermind.Runner/configs/mainnet_archive.cfg

```diff
@@ -7,7 +7,7 @@
     "MemoryHint": 4096000000
   },
   "Network": {
-    "ActivePeersMaxCount": 100
+    "ActivePeersMaxCount": 50
```

Comment:
If this is default, remove it altogether!

Reply:
Done